packages/sandcastle/src/App.tsx

```diff
@@ -639,9 +677,19 @@ Sandcastle.addToolbarMenu(${variableName});`,
     window.focus();
   }
 
-  function loadDemo(demo: GalleryDemo) {
-    // do stuff
-    setCode(demo.js ?? defaultJsCode, demo.html ?? defaultHtmlCode);
+  const GALLERY_BASE = "/gallery";
```

Comment:
I _think_ that doing something like this would allow us to point it at literally any path as long as it conforms to the structure we'd expect. Namely that there's a `list.json` at the root and `[slug]/(index.html|main.js)` files. This could help us swap out where it points for different local/deployed builds or even potentially a different server entirely if someone wants to host their own gallery?